Makefile.dep

```diff
@@ -606,6 +606,14 @@ ifneq (,$(filter random,$(USEMODULE)))
   endif
 endif
 
+ifneq (,$(filter random_seed,$(USEMODULE)))
+  USEMODULE += luid
+  USEMODULE += xtimer
+
+  FEATURES_REQUIRED += periph_hwrng
+  FEATURES_REQUIRED += periph_adc
```

Comment:
What happens if you use `FEATURES_OPTIONAL` and then check in the implementation which ones are actually included?